tests/test_datasetattributes.py

```diff
@@ -197,6 +204,16 @@ def test_set_invalid_vectors(hexbeam):
         hexbeam.point_data.set_vectors(not_vectors, 'my-vectors')
 
 
+def test_set_tcoords_name():
+    mesh = pyvista.Cube()
+    old_name = mesh.point_data.active_t_coords_name
+    assert mesh.point_data.active_t_coords_name is not None
+    mesh.point_data.active_t_coords_name = None
+
+    mesh.point_data.active_t_coords_name = old_name
+    assert mesh.point_data.active_t_coords_name == old_name
```

Comment:
Shouldn't we check in between that it was set to `None`?

Reply:
```suggestion
    old_name = mesh.point_data.active_t_coords_name
    assert mesh.point_data.active_t_coords_name is not None
    mesh.point_data.active_t_coords_name = None
    assert mesh.point_data.active_t_coords_name is None

    mesh.point_data.active_t_coords_name = old_name
    assert mesh.point_data.active_t_coords_name == old_name
```